src/core/featurelistmodel.cpp

```diff
@@ -280,6 +284,14 @@ QgsFeature FeatureListModel::getFeatureFromKeyValue( const QVariant &value ) con
   return feature;
 }
 
+QgsFeature FeatureListModel::getFeatureFromId( QgsFeatureId id ) const
```

Comment:
```suggestion
QgsFeature FeatureListModel::getFeatureById( QgsFeatureId id ) const
```

I think the convention is usually "get by", instead of "get from"?